operator/src/test/java/oracle/kubernetes/operator/MainTest.java

```diff
@@ -833,10 +836,66 @@ public void withNamespaceList_onReadExistingNamespaces_whenDomainNamespaceRemove
     HelmAccessStub.defineVariable(HelmAccess.OPERATOR_DOMAIN_NAMESPACES, namespaceString);
     createNamespaces(4);
 
-    testSupport.runSteps(createDomainRecheck().readExistingNamespaces());
+    runCreateReadNamespacesStep();
     assertThat(logRecords, containsInfo(MessageKeys.END_MANAGING_NAMESPACE, "NS3"));
   }
 
+  @Test
+  public void withNamespaceList_changeToDedicated_onCreateReadNamespaces_nsWatchStoppedEventCreated() {
+    setupDomainWithNamespaceListStrategy("NS1");
+    runCreateReadNamespacesStep();
+    defineSelectionStrategy(SelectionStrategy.Dedicated);
+    runCreateReadNamespacesStep();
+
+    assertThat("Found NAMESPACE_WATCHING_STOPPED event with expected message",
+        containsEventWithMessageForNamespaces(getEvents(testSupport),
+            NAMESPACE_WATCHING_STOPPED, Collections.singletonList("NS1")), is(true));
+  }
+
+  @Test
+  public void withNamespaceList_changeToDedicated_onCreateReadNamespaces_nsWatchStartedEventCreatedInOpNS() {
+    setupDomainWithNamespaceListStrategy("NS1");
+    runCreateReadNamespacesStep();
+    defineSelectionStrategy(SelectionStrategy.Dedicated);
```

Comment:
Here you are changing to a new strategy, so line 859 begins the 'action' part of the test and should be separated by a blank line from the 'setup'